### DISCUSSION TITLE
Valheim 80% cpu usage

### DISCUSSION THREAD

**Discussion by rjpiston** [GameServerManagers/LinuxGSM#3247] [General]
First time using linuxgsm, everything is working great so far. I've noticed though, that my cpu usage on the server running vhserver is around 80% constantly, even when no one is logged in. Was just wondering if this is normal, or if there is a start parameter I can use to limit the cpu usage? 

I'm running ubuntu 20.04, i5 4690k.

Thanks

**Comment by dgibbs64** [comment]
I have one core at around 60%. So possibly yes. My guess is that it's not fully optimised yet,

**Reply by rjpiston** [reply]
fair enough, just wanted to make sure I wasn't the only one. CPU temps seem fine and the server has been super stable so far. Thanks for the reply
